capstone-ui/src/Components/PresidentStatePage/PresidentStatePage.jsx

```diff
@@ -0,0 +1,18 @@
+import * as React from "react"
+
+export default function PresidentialStatePage({ stateName }) {
+    return (
+        <div className="state-page">
+            {console.log(stateName)}
```

Comment:
Let's remove this `console.log`

Reply:
Don't know why it got pushed with that in there. It's not there anymore, so I'm assuming I must've erased it at some point after.